tree/ntuple/v7/inc/ROOT/RNTupleWriter.hxx

```diff
@@ -110,6 +110,8 @@ public:
    /// Fill an entry into this ntuple, but don't commit the cluster. The calling code must pass an RNTupleFillStatus
    /// and check RNTupleFillStatus::ShouldCommitCluster.
    void FillNoCommit(REntry &entry, RNTupleFillStatus &status) { fFillContext.FillNoCommit(entry, status); }
+   /// Flush column data, preparing for CommitCluster or to reduce memory usage
```

Comment:
We might want to qualify and/or expand on the meaning of `Flush` (eg. some reader might thing to this flushing to the underlying storage hardware - the later part of the description hints that it is not the case)

Reply:
I amended the doc to say that `[this] will generally trigger compression of pages, but not actually write to storage (unless buffered writing is turned off).`